scripts/beaker/tiny/torchrun-script.sh

```diff
@@ -26,9 +26,10 @@ torchrun \
   --rdzv_backend=c10d \
   --rdzv_endpoint=$BEAKER_LEADER_REPLICA_HOSTNAME:29400 \
   scripts/train.py \
-    configs/tiny/OLMo-30M.yaml \
+    configs/tiny/OLMo-20M.yaml \
       --run_name=$TASK_NAME \
       --wandb.name=$TASK_NAME \
       --wandb.group=$TASK_NAME \
-      --wandb.project=tiny_olmo \
+      --wandb.project=olmo-tiny \
+      --max_grad_norm=2.0 \
```

Comment:
Do you want to do this clipping value for all small models?

Reply:
ah, let me fix this, so the model with clipping value 2.0 does not show any downstream improvement!